resources/qml/delegates/Encrypted.qml

```diff
@@ -4,45 +4,71 @@
 
 import ".."
 import QtQuick 2.15
-import QtQuick.Controls 2.1
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
 import im.nheko 1.0
 
-Column {
+Rectangle {
     id: r
 
     required property int encryptionError
     required property string eventId
 
+    radius: height / 4
```

Comment:
This should probably be a fixed value, so that when the window is narrow, it doesn't clip the text. Probably fontMetrics.lineHeight + paddingMedium